WalletWasabi.Fluent/ViewModels/CoinJoinProfiles/CoinJoinProfilesViewModel.cs

```diff
@@ -46,6 +46,8 @@ public static CoinJoinProfileViewModelBase IdentifySelectedProfile(KeyManager ke
 		var currentProfile = new ManualCoinJoinProfileViewModel(keyManager);
 		var result = DefaultProfiles.FirstOrDefault(x => x == currentProfile) ?? currentProfile;
 
+		result.AnonScoreTarget = currentProfile.AnonScoreTarget;    // Edge case: Update the PrivateCJProfile anonscore target, otherwise the randomly selected value will be displayed all time.
```

Comment:
probably better to move the comment on a full line

Reply:
Done [6742c30](https://github.com/zkSNACKs/WalletWasabi/pull/9593/commits/6742c308c12922566585ef61b0a849b1d54c3d4c)